Documentation/index.md

```diff
@@ -0,0 +1,28 @@
+Documents Index
+===
+
+This repo includes several documents that explain both high-level and low-level concepts about the .NET runtime. These are very useful for contributors, to get context that can be very difficult to acquire from just reading code.
+
+# Book of the Runtime
+
+- [Introduction to the Common Language Runtime](intro-to-clr.md)
+- [Mscorlib and Calling Into the Runtime](mscorlib.md)
+- [Stack Walking](stackwalking.md)
+- [Book of the Runtime FAQ](botr-faq.md)
+
+# Coding Guides
+- [CLR Coding Guide](clr-code-guide.md)
+
+# Glossary
+- [.NET Core Glossary](glossary.md)
+- [.NET Filename Encyclopedia](dotnet-filenames.md)
+
+# Links to other sources
+
+- [MSDN Entry for the CLR][clr](http://msdn.microsoft.com/en-us/library/8bs2ecf4(VS.71).aspx)
```

Comment:
Looks like an additional `[clr]` slipped in here that breaks rendering of the link.